main.py

```diff
@@ -1,10 +1,46 @@
 import init_django_orm  # noqa: F401
 
+import json
+from django.utils.timezone import now
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+
+    for player_name, player_info in data.items():
+        race_data = player_info["race"]
+        race, _ = Race.objects.get_or_create(
+            name=race_data["name"],
+            defaults={"description": race_data["description"]}
+        )
+
+        for skill_data in race_data["skills"]:
+            Skill.objects.get_or_create(
+                name=skill_data["name"],
+                defaults={"bonus": skill_data["bonus"], "race": race}
+            )
+
+        guild_data = player_info["guild"]
+        guild = None
+        if guild_data:
+            guild, _ = Guild.objects.get_or_create(
+                name=guild_data["name"],
+                defaults={"description": guild_data["description"]}
+            )
+
+        Player.objects.get_or_create(
```

Comment:
This is not an error but a suggestion for the future. It's a good practice to separate the logic of creating objects into different functions. It will make your code more readable and maintainable. For example, you could create separate functions for creating a Race, Skill, Guild, and Player.